extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.text-formatter.js

```diff
@@ -57,32 +61,75 @@ define('ext.wikia.design-system.on-site-notifications.text-formatter', [
 			};
 
 			this._getReplyKey = function (title, totalUniqueActors) {
-				var user = totalUniqueActors <= 1 ? '' :
```

Comment:
These changes to getting the i18n key for replies and upvotes is unrelated to the main work of this ticket, but building the keys like this piecemeal makes them really hard to grep for and, IMO, code should be easily grepable. So I updated the logic so we're using their complete names instead of building them up using their common parts.